vmdb/db/migrate/20150630025128_create_tenant.rb

```diff
@@ -0,0 +1,17 @@
+class CreateTenant < ActiveRecord::Migration
+  def change
+    create_table :tenants do |t|
+      t.string :domain
+      t.string :subdomain
+      t.string :name
+      t.string :vmdb_name
```

Comment:
I'm just using the current session: [session assignment]
I can change to `current_tenant.name` to `current_tenant.company_name` if you would like.

[session assignment]: https://github.com/ManageIQ/manageiq/pull/3336/files#diff-581a01e4cf7ebb71fa38f17b1b22339aR10